src/bundle/Resources/public/scss/_modals.scss

```diff
@@ -95,3 +95,37 @@
         background-color: $ez-ground-base;
     }
 }
+
+.ez-modal--version-draft-conflict {
+    .modal-header {
+        background: $ez-color-base-pale;
+        border-bottom: none;
+
+        .modal-title {
+            font-size: 1.5rem;
+        }
+
+        .close {
+            margin-top: -.5625rem;
+        }
+    }
+
+    .modal-body {
+        padding: 2rem;
+        background-color: $ez-ground-base-medium;
+        border-radius: 0 0 .25rem .25rem;
+
+        .ez-modal-body__main {
+            margin-bottom: 0;
+        }
+
+        .ez-modal-body__main:last-of-type {
```

Comment:
Is there a possibility that there are many elements with `.ez-modal-body__main` inside `.modal-body`?

Reply:
Right now it's two sentences only, but it might be changed when translated or adapted when extending it. How do you see it?

Reply:
This class should be added to only one element inside the modal. Because it's meant to be used with the main content of the modal. I can't imagine someone is using it twice, because it will be structuraly incorrect.